modules/ROOT/pages/vm/vm-publish-listen.adoc

```diff
@@ -1,25 +1,27 @@
-= Example: To Publish and Listen to Messages through the VM Connector - Mule 4
+= Example: Publish and Listen to Messages through the VM Connector - Mule 4
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
 
-This example shows how to publish a message that is received by a different flow.
+The following example shows how to publish a message that is received by a different flow.
+
+. Define a flow with a `<vm:listener>` as a message source.
+
+The listener flow will receive the published message from a different flow, for example:
```

Comment:
"The listener flow receives the published message from the yourIntegrationLogic flow:"